src/test/java/org/radarbase/ksql/udaf/UdafTest.java

```diff
@@ -0,0 +1,100 @@
+package org.radarbase.ksql.udaf;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.radarbase.ksql.udaf.UniformSamplingReservoirUdaf.COUNT;
+import static org.radarbase.ksql.udaf.UniformSamplingReservoirUdaf.SAMPLES;
+
+import io.confluent.ksql.function.udaf.Udaf;
+import java.util.ArrayList;
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.SchemaBuilder;
+import org.apache.kafka.connect.data.Struct;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class UdafTest {
+
+    private static final Double[] values = new Double[]{40.0, 50.0, 60.0};
```

Comment:
Perhaps add a test where you initialise the `maxSize` of the Udaf to be 2? Or a test where you add >=10.000 samples?